components/common/SocialLinks.tsx

```diff
@@ -47,7 +47,7 @@ interface SocialLinksProps {
 const SocialLinks: FC<SocialLinksProps> = ({ aboutPage }): ReactElement => {
   const wrapperStyle = aboutPage
```

Comment:
` const wrapperStyle = aboutPage
   ? 'grid grid-cols-3 gap-x-8 gap-y-3 largeTablet:flex'
   : 'flex flex-wrap relative gap-x-3 pr-2'
`

Why do we need a specific style only for the about page? What's the difference between the about page and all other pages?

Reply:
Noticed that also but that was how I met it in the repo